android/capacitor/src/main/java/com/getcapacitor/FileUtils.java

```diff
@@ -289,4 +290,14 @@ private static String getPathToNonPrimaryVolume(Context context, String tag) {
         }
         return null;
     }
+
+    private String sanitizeFilename(String displayName) {
```

Comment:
```suggestion
    private static String sanitizeFilename(String displayName) {
```
method needs to be `static` since it's called from `getCopyFilePath`, which is `static`

Reply:
Good spot. Thanks. I've made that change.